src/main/kotlin/com/openlattice/data/ids/PostgresEntityKeyIdService.kt

```diff
@@ -82,15 +88,53 @@ class PostgresEntityKeyIdService(
         return entityKeyIds
     }
 
-    override fun reserveEntityKeyIds( entityKeys: Set<EntityKey> ): Set<UUID> {
-        val entityIdsByEntitySet = entityKeys.groupBy ({ it.entitySetId },{it.entityId}).mapValues { it.value.toSet() }
+    private fun countUpEntityKeys(entityKeys: Set<EntityKey>) {
+        idRefCounts.executeOnKeys(entityKeys, IdRefCountIncrementer())
```

Comment:
Do we ever add entries to idRefCounts?

Reply:
Entry processor can add entries through set value

Reply:
Yeah, but if there are no matching keys, will it still execute?

Reply:
Yes.

Reply:
Ah, okay! Thanks for clarifying that!